lib/shopify_transporter/exporters/exporter.rb

```diff
@@ -18,15 +18,14 @@ def initialize(config_filename, object_type)
       end
 
       def run
-        data = Magento::MagentoExporter
+        Magento::MagentoExporter
           .for(type: object_type)
           .new(
             store_id: config['export_configuration']['store_id'],
             soap_client: soap_client,
             database_adapter: database_adapter
           )
           .export
```

Comment:
Maybe there's a better name for these functions now? They aren't just exporting anymore -- they're printing data to standard out.